utils/css.ts

```diff
@@ -57,6 +66,10 @@ export function serializeCSS(
       )
     }
 
+    if (typeof scale === 'number' && scale !== 1 && !useRem) {
```

Comment:
```suggestion
    if (typeof scale === 'number' && scale !== 1) {
```

I think `useRem` should be independent with `scale`.

Reply:
Get